ui/apps/platform/src/types/reportConfigurationService.proto.ts

```diff
@@ -51,24 +51,17 @@ type ScheduleBase = {
 };
 
 export type Schedule =
-    | (ScheduleBase & {
-          weekly: WeeklyInterval;
-      })
     | (ScheduleBase & {
           daysOfWeek: DaysOfWeek;
       })
     | (ScheduleBase & {
           daysOfMonth: DaysOfMonth;
       });
 
-export type IntervalType = 'UNSET' | 'DAILY' | 'WEEKLY' | 'MONTHLY';
+export type IntervalType = 'WEEKLY' | 'MONTHLY';
```

Comment:
The API docs mentioned that there was a `DAILY` option but the current implementation doesn't have it and looks like we aren't going to go that route anymore